e2e-tests/spring-rest-openapi-v3/em.yaml

```diff
@@ -0,0 +1,89 @@
+### Template configuration file for EvoMaster.
```

Comment:
why adding this to the repository? was it a mistake?

Reply:
yes. somehow missed it.